ui/serve.go

```diff
@@ -31,19 +42,22 @@ func Register(r *gin.Engine, version model.VersionInfo, register bool) {
 	ui.GET("/index.html", serveFile("index.html", "text/html", noop))
 	ui.GET("/manifest.json", serveFile("manifest.json", "application/json", noop))
 	ui.GET("/asset-manifest.json", serveFile("asset-manifest.json", "application/json", noop))
-	ui.GET("/static/*any", gin.WrapH(http.FileServer(box)))
+	ui.GET("/static/*any", gin.WrapH(&AddPrefix{
+		Prefix:       "/build",
+		InnerHandler: http.FileServer(http.FS(box)),
+	}))
```

Comment:
Please use [fs.Sub](https://pkg.go.dev/io/fs#Sub) instead of the custom AddPrefix
```
//go:embed build/*
var embedFS embed.FS
var buildUI, _ = fs.Sub(embedFS, "build")

func Register() {
	ui.GET("/static/*any", gin.WrapH(http.FileServer(http.FS(buildUI))))
}
```